src/Sylius/Component/Core/spec/Model/TaxonomySpec.php

```diff
@@ -25,7 +31,7 @@ function it_is_Sylius_Taxonomy()
         $this->shouldImplement('Sylius\Component\Taxonomy\Model\TaxonomyInterface');
     }
 
-    function it_should_not_path_defined_by_default()
+    function it_should_not_have_path_defined_by_default()
     {
         $this->getRoot()->getPath()->shouldReturn(null);
     }
```

Comment:
I'm not one of them (:smile:), but I think it's because of missing ``TaxonomyFactory`` class name parameter in ``services.xml`` file ;)